tests/test_deregister_task_definition.py

```diff
@@ -0,0 +1,232 @@
+import unittest
+from typing import Literal
+from unittest.mock import Mock, call, patch
+
+import boto3
+
+from actions_helper.commands.deregister_task_definition import deregister_task_definition
+from actions_helper.outputs import CreateTaskDefinitionOutput
+
+
+class DeregisterTaskDefinitionTestCase(unittest.TestCase):
+    @patch.object(boto3, "client")
+    def setUp(self, boto3_client) -> None:
+        self.ecs_client = boto3_client
+
+    @staticmethod
+    def describe_service_return_value(status: Literal["ACTIVE", "INACTIVE", "PRIMARY"], arn: str):
+        return {"services": [{"deployments": [{"status": status, "taskDefinition": arn}]}]}
+
+    def test_deregister_task_definition_no_task_definitions(self):
+        with (
+            patch.object(self.ecs_client, "deregister_task_definition") as ecs_deregister_patch,
+            patch.object(
+                self.ecs_client,
+                "describe_services",
+                return_value=self.describe_service_return_value("PRIMARY", arn=""),
```

Comment:
```suggestion
                return_value=self.describe_service_return_value(status="PRIMARY", arn=""),
```
Please also change that in other places where `describe_service_return_value()` is called.